data/mods/My_Sweet_Cataclysm/sweet_comestibles.json

```diff
@@ -151,5 +151,28 @@
     "calories": 152,
     "description": "Dehydrated chocolate milk powder.  Mix with water to make drinkable milk.",
     "fun": -1
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "licorice",
+    "name": "licorice",
+    "symbol": "%",
+    "volume": "40 ml",
+    "weight": "78 g",
+    "spoils_in": "230 days",
+    "rot_spawn": "GROUP_EGG_LICORICE",
+    "rot_spawn_chance": 100,
+    "color": "red",
+    "comestible_type": "FOOD",
+    "material": [ "junk" ],
+    "description": "A swirly confectionary treat in strings of sweet tasting candy.",
```

Comment:
Add a short thing at the end of the description to suggest that something is not right wit this candy. See other candy in MSC for exemples.

Reply:
Added more to the description